### ISSUE TITLE
datadog-for-the-mendix-cloud

### ISSUE THREAD

**Issue by joepveenman** [mendix/docs#6501]
Please use the form below, leaving the prefilled data to help us. Thank you.

Page link: [datadog-for-the-mendix-cloud](https://docs.mendix.com/developerportal/operate/datadog-metrics/)

Document link: [datadog-metrics.md](https://github.com/mendix/docs/blob/development/content/en/docs/developerportal/operate/metrics/monitoring-with-apm/datadog-metrics.md)

## My Issue/Suggestion

Dear Mendix,
There is on the page/document mentioned above the following text:
"The DD_LOG_LEVEL sets the level for which log messages from the Mendix Datadog agent will be sent to the Mendix application logs. It does not affect the [log level set in your app](https://docs.mendix.com/howto/monitoring-troubleshooting/log-levels/). Valid values are: CRITICAL, ERROR, WARNING, INFO, DEBUG"

I believe that DD_LOG_LEVEL is meant to work as a filter for witch log are sent from Mendix to datadog. That is what https://docs.mendix.com/developerportal/deploy/environments-details/ states and what would be logical.
But in this text they talk about "Datadog agent will be sent to the Mendix". What is kind of weird. Why would data dog sent anything to Mendix.

If DD_LOG_LEVEL is a filter from Mendix to datadog, Then it is not working for us. 
Even with DD_LOG_LEVEL set to WARNING is it still sending INFO messages to our data dog server. 
This is very sad because we can go live right now because of that.

To summarize we are trying to reduce the amount of messages sent to datadog to only the massages with level: CRITICAL, ERROR or WARNING. Setting DD_LOG_LEVEL to WARNING or ERROR did not help. What DD_LOG_LEVEL should do isn't clear to us neither clear to us how we can reduce our datadog messages.

thank you for your time.

**Comment by MarkvanMents** [comment]
Hi @joepveenman,
Sorry about the delay in getting back to you.
I have checked with the developers and they have confirmed that **DD_LOG_LEVEL** only impacts the messages which come from the Datadog agent. As stated in the text, it has no impact on the log level set in your app. So if your app is generating INFO log messages these will not be filtered by the Datadog agent and you should adjust the settings within your app to filter messages you don't want to see.

I imagine that this setting allows you to get INFO and DEBUG messages from the Datadog agent if you need to trace something unexpected with what is being sent to Datadog.

The documentation is obviously not clear enough and I am discussing with the developers how I can make this clearer.

I hope this helps. If you have further questions, can I suggest the [Mendix Forum](https://forum.mendix.com/p/questions) where the Mendix community may be able to help from their own experiences.

Mark

**Comment by joepveenman** [comment]
Hi Mark,
Thank you very much for your reaction. This did clear things up a lot.

Joep
